File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        
        # a = str(x)
        
        # p = a[::-1]
        
        # if a == p:
        #     return True
        # else:
        #     return False
        a = str(x)
        p = a[::-1]
        if(a == p):
            return True
        else:
            return False

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x<0) return false;
        int ans = 0;
        int n = x;
        while(x>0){
            int digit = x%10;
            ans = ans*10+digit;
        x = x/10;
        }
        if (ans == n) return true;
        return false;
    }
}